Makefile

```diff
@@ -52,3 +54,11 @@ dep-upgrade-all:
 
 diff:
 	git diff --color --ignore-all-space --ignore-blank-lines --exit-code
+
+test: generate build-test
+	go test -v ./...
+	echo "Running inperoperability test"; \
+		./bin/webrpc-test -server -port=9988 -timeout=2s & \
+		sleep 0.5; \
+		./bin/webrpc-test -client -url=http://localhost:9988; \
```

Comment:
This runs inperoperability test (client <-> server) using `golang@latest` target